models/Version.php

```diff
@@ -226,41 +236,32 @@ public function save()
             $dataString = $data;
         }
 
-        $isAssetFile = false;
-        if ($data instanceof Asset && $data->getType() != 'folder') {
-            $isAssetFile = true;
-
+        if($data instanceof Asset) {
```

Comment:
why `$data->getType() != 'folder'` was removed as condition there?

Reply:
Added the type check again